src/main/target/common_pre.h

```diff
@@ -356,13 +356,11 @@
 #if (TARGET_FLASH_SIZE > 256)
 #define USE_AIRMODE_LPF
 #define USE_CANVAS
-#define USE_DASHBOARD
```

Comment:
The dashboard is the statics info after disarming a flight right? Maybe useful for ppl to have a basic idea how things going on.